packages/core/demo/data/alluvial.ts

```diff
@@ -25,6 +25,18 @@ export const alluvialSimpleOptions = {
 	},
 };
 
+export const alluvialSimpleCustomColorOptions = {
+	...alluvialSimpleOptions,
+	title: 'Custom colors (alluvial)',
+	color: {
+		scale: {
+			A: 'red',
+			B: 'yellow',
+			C: 'blue',
```

Comment:
Could we pls just tone down the colors a little bit?

Reply:
you could even use the palette https://www.carbondesignsystem.com/data-visualization/color-palettes/

Reply:
<img width="747" alt="image" src="https://user-images.githubusercontent.com/38994122/136075546-6aba469a-a9a9-4463-be76-67c3b877b83d.png">